Doc/library/ssl.rst

```diff
@@ -2603,7 +2603,7 @@ with LibreSSL.
        Documentation of underlying :mod:`socket` class
 
    `SSL/TLS Strong Encryption: An Introduction <https://httpd.apache.org/docs/trunk/en/ssl/ssl_intro.html>`_
-       Intro from the Apache webserver documentation
+       Intro from the Apache web server documentation
```

Comment:
Perhaps "Apache HTTP Server" would be a better choice than "Apache web server"? From their website at https://httpd.apache.org/

> The Apache HTTP Server ("httpd") was launched in 1995 and it has been the most popular web server on the Internet since April 1996.

Reply:
I agree.  Will send an update now.